src/sas/qtgui/Perspectives/Fitting/FittingWidget.py

```diff
@@ -3878,60 +3878,95 @@ def getParameterDict(self):
                 line_dict[content[0]] = content[1:]
         return line_dict
 
-    def onCopyToClipboard(self, format=None):
+    def copy(self):
```

Comment:
I don't think this naming is explanatory.
If I see `widget.copy()` I expect this to return me a shallow copy of the widget object.
Maybe `copyToClipbard` is a better name, after all

Reply:
Likewise, other methods, which names you modified.